ignite-generator/component/templates/component-test.js.template

```diff
@@ -0,0 +1,24 @@
+import test from 'ava'
+import React from 'react'
+import { shallow } from 'enzyme'
+import <%= name %> from '../../App/Components/<%= name %>'
+
+test('component exists', t => {
+  const wrapper = shallow(<<%= name %>  />)
```

Comment:
one thing I've started doing is moving these `const`s to above all functions, so I don't have to repeat `wrapper` often.